packages/cli/src/config/extension-manager.ts

```diff
@@ -136,11 +140,19 @@ export class ExtensionManager implements ExtensionLoader {
     let extension: GeminiCLIExtension | null;
     try {
       if (!isWorkspaceTrusted(this.settings).isTrusted) {
```

Comment:
This seems to be checking if the workspace is trusted but the error is logging the extension source. I don't know which is correct, but I think we should figure that out prior to landing this.